jekyll/_includes/sidebar.html

```diff
@@ -12,7 +14,7 @@
 						{% endif %}
 						
 						{% if main-item.link %}
-							<a class="{% if page.url == main-item.link %} active {% endif %}" href="{{ site.baseurl }}/{{ main-item.link }}#section={{ main-item.name | slugify }}" data-proofer-ignore>{{ main-item.name}}</a>
+							<a class="{% if current_url == main-item.link %} active {% endif %}" href="{{ site.baseurl }}/{{ main-item.link }}#section={{ main-item.name | slugify }}" data-proofer-ignore>{{ main-item.name}}</a>
```

Comment:
The problem here is that each one of these would require a different `isActive` declaration, since the item being compared to is unique to the scope of each loop. 

I could still add it to improve the semantics, but it wouldn't be much of a space/flow improvement.